tests/calc/test_thermo.py

```diff
@@ -201,8 +200,9 @@ def test_moist_lapse_starting_points(start, direction):
 @pytest.mark.xfail(platform.machine() == 'aarch64',
                    reason='ValueError is not raised on aarch64')
 @pytest.mark.xfail(platform.machine() == 'arm64', reason='ValueError is not raised on Mac M2')
-@pytest.mark.xfail(sys.platform == 'win32', reason='solve_ivp() does not error on Windows')
-@pytest.mark.xfail(packaging.version.parse(scipy.__version__) < packaging.version.parse('1.7'),
+@pytest.mark.xfail((sys.platform == 'win32') & version_check('scipy<1.11.3'),
```

Comment:
Unless we're working with arrays, better to use logical, rather than bitwise, and:
```suggestion
@pytest.mark.xfail((sys.platform == 'win32') and version_check('scipy<1.11.3'),
```

Reply:
Oh woops, thanks